### PR TITLE
Capstone Project Completion (PR)

### PR DESCRIPTION
**Pull request by Doheera-kosi**
## After completing all of the assignments of separate branches and merging each of them to the dev branch, we are ready to merge to the main.

- Fetch data from mealDBApi
- Fetch data from InvolvementApi
- Create comment
- Give a like button
- Create a test for counting elements in the homepage and the number of comments.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,81 @@
...
+### To get a local copy up and running follow these simple example steps.
+
+- Clone the repo to your local environment
+- Run npm install to install all dependencies
+- Open the index.html file with your favorite browser.
+- Modify to your taste with your preferred text editor
+- Enjoy 🤗
+
+
+
```
Comment (BertrandConxy) [inline README.md]: - [ ] Good job so far 👏 Your README looks fine but as your web app uses 2 APIs, It is essential to provide enough information about it. For the case of the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270) you know someone needs to have the key. Therefore, you should state that also here in this section. In short, please try to provide information about the APIs that your app consumes and how to use them 😃 👍

File: src/modules/likes.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    {
+      method: 'POST',
+      headers: { 'Content-type': 'application/json' },
+      body: JSON.stringify({
+        item_id: id,
+      }),
+    },
+  );
+};
+// https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/eJ0dYt3woHnL6Sz1FyVG/likes
```
Comment (BertrandConxy) [inline src/modules/likes.js]: - [ ] Please **remove these commented codes in order to make your codebase clean** 😉 👍

File: src/modules/popup.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+      </form>
+
+    </div>
+  </div>
+`;
+      fetchComments(cat.idCategory);
+      createComment(cat.idCategory);
+    }
+  });
+  // const comList = document.querySelector('#comment_list');
```
Comment (BertrandConxy) [inline src/modules/popup.js]: - [ ] please **remove these commented codes** as they make your codebase unclean 👍

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hello @Doheera-kosi, 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 

## Highlights 💪 
- No linter errors
- correct flow is used
- good PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/likes.js (hunk 2 above)
Comment (Doheera-kosi) [reply src/modules/likes.js]: Hi, @BertrandConxy
Thank you for the feedback. We have removed the comments code!

File: src/modules/popup.js (hunk 3 above)
Comment (Doheera-kosi) [reply src/modules/popup.js]: Hi, @BertrandConxy
Thank you for the feedback. We have removed the comments code!

Comment (Doheera-kosi) [reply src/modules/popup.js]: Hi, @BertrandConxy
Thank you for the feedback. We have removed the comments code!

File: src/modules/likes.js (hunk 2 above)
Comment (Doheera-kosi) [reply src/modules/likes.js]: Hi, @BertrandConxy
Thank you for the feedback. We have removed the comments code!

File: README.md (hunk 1 above)
Comment (Doheera-kosi) [reply README.md]: Hi, @BertrandConxy
Thank you for the feedback. We have added the APIs links for the Project!

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle:  🎉 🎉 🎉
Hi @Doheera-kosi & @teke85 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights :heavy_check_mark: 
- [ ] Nice design :white_check_mark: 
- [ ] Good commit messages :white_check_mark: 
- [ ] Descriptive pull request title and summary :white_check_mark: 
- [ ] Gitflow is used correctly :white_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**